client/src/modules/privacy-policy/components/PrivacyPolicy.en.md

```diff
@@ -0,0 +1,233 @@
+# PRIVACY POLICY
+
+_Last updated July 25, 2019_
+
+Thank you for choosing to be part of our community at **Sandstream Development Sp. z o.o.** ("company", "we", "us", or "our"). We are committed to protecting your personal information and your right to privacy. If you have any questions or concerns about our policy, or our practices with regards to your personal information, please contact us at [eoc.sandstream@gmail.com](mailto:eoc.sandstream@gmail.com).
+
+When you visit our website [app.eoc.sanddev.com](https://app.eoc.sanddev.com), mobile application, and use our services, you trust us with your personal information. We take your privacy very seriously. In this privacy notice, we describe our privacy policy. We seek to explain to you in the clearest way possible what information we collect, how we use it and what rights you have in relation to it. We hope you take some time to read through it carefully, as it is important. If there are any terms in this privacy policy that you do not agree with, please discontinue use of our Sites or Apps and our services.
```

Comment:
`please discontinue the use of our Sites or Apps and our services.`    use -> the use